client/app/components/Link.tsx

```diff
@@ -0,0 +1,22 @@
+import React
+ from "react";
```

Comment:
pastram importurile pe o sg line 

```suggestion
import React from "react";
```